test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/ServiceBusEndToEndTests.cs

```diff
@@ -235,6 +235,8 @@ private async Task ServiceBusEndToEndInternal<T>(IHost host = null)
                 string[] consoleOutputLines = consoleOutput
                     .Where(p => p.FormattedMessage != null)
                     .SelectMany(p => p.FormattedMessage.Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries))
+                    // There is a known issue in SB SDK 4.0.0 in MessageReceivePump.MessagePumpTaskAsync https://github.com/Azure/azure-sdk-for-net/issues/6410
+                    .Where(p => !p.StartsWith("Message processing error"))
```

Comment:
How prevalent is this error? Will this cause a lot of errors logged for customers (and thus CRIs)? Looks like they've fixed it -- should we wait for their next release to move forward?

Reply:
Make swnse. I will delay the release.